### ISSUE TITLE
Lumping support for multi-word prepositions?

### ISSUE THREAD

**Issue by videophonegeek** [nlp-compromise/nlp_compromise#247]
**e.g. 2 word prepositions:**
according to
adjacent to
ahead of
apart from
as for
as of
as per ...
**e.g. 3 word prepositions:**
as well as
at the behest of
by means of
by virtue of
for the sake of ...

The single term prepositions are recognized but not the multi-word. 
![prepositions](https://cloud.githubusercontent.com/assets/191271/17075570/7cb8dce2-50d2-11e6-8f11-c292ff4f0cf3.PNG)

**Comment by spencermountain** [comment]
hi John! this is a good point. I think we should support a bunch of these examples. Do you have more? I think most of them are good candidates to drop into the lexicon straight-up.
I guess it's no longer a preposition if it conjugates? 'adjacent to' works, but 'close to' doesn't?
happy to add these!
cheers

**Comment by videophonegeek** [comment]
Yes, I have more examples. Here is a snippet from my data showing the two and three word prepositions and also attached file list of prepositions 1, 2, and 3:

according to
adjacent to
ahead of
apart from
as for
as of
as per
as regards
aside from
astern of
back to
because of
close to
due to
except for
far from
inside of
instead of
left of
near to
next to
opposite of
opposite to
out from
out of
outside of
owing to
prior to
pursuant to
rather than
regardless of
right of
subsequent to
such as
thanks to
up to
as far as
as opposed to
as soon as
as well as
at the behest of
by means of
by virtue of
for the sake of
in accordance with
in addition to
in case of
in front of
in lieu of
in order to
in place of
in point of
in spite of
on account of
on behalf of
on top of
with regard to
with respect to
with a view to

[prepositions.txt](https://github.com/nlp-compromise/nlp_compromise/files/379356/prepositions.txt)

**Comment by spencermountain** [comment]
thanks, i'm just working on a [pretty big re-write](https://github.com/nlp-compromise/nlp_compromise/blob/v7/README.md). will add these in once things are stable.
in the meantime, you can [augment the lexicon](https://github.com/nlp-compromise/nlp_compromise/wiki/Lexicon#adding-words), as you wish.
cheers

**Comment by videophonegeek** [comment]
It was easy to roll out my (less developed) NLP and use NLP-compromise.  As you have seen, I have a little GUI for interface so easy to test. It also works from speech to text and I am trying to integrate recognition of homonyms and oronyms to improve interpretation of results. As  you type it does an update of terms when you type a space or hit enter. Look forward to your next release and ready to test.

**Comment by spencermountain** [comment]
very neat!
